<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstStart</class>
 <widget class="QDialog" name="firstStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/menu/Help_AboutOpenAtlas</normaloff>:/menu/Help_AboutOpenAtlas</iconset>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Welcome to openATLAS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>480</width>
     <height>190</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>190</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>190</height>
    </size>
   </property>
   <property name="title">
    <string>Step 1</string>
   </property>
   <widget class="QLineEdit" name="lineEditPathConnections">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonOpenFileDialog">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>30</y>
      <width>33</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/buttons/folder</normaloff>:/buttons/folder</iconset>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPathQuestion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_01_Cancel">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/cancel</normaloff>:/buttons/cancel</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_01_Next">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/forward</normaloff>:/buttons/forward</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_02_Back">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/reward</normaloff>:/buttons/reward</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_02_Next">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/forward</normaloff>:/buttons/forward</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_03_Next">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/forward</normaloff>:/buttons/forward</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_01_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>firstStart</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
